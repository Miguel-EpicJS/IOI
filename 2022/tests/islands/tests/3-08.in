4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
100 198
34 22
22 34
30 81
81 30
6 18
18 6
74 40
40 74
36 9
9 36
24 89
89 24
15 72
72 15
21 20
20 21
39 95
95 39
9 16
16 9
25 68
68 25
45 7
7 45
84 50
50 84
54 69
69 54
13 31
31 13
47 73
73 47
26 87
87 26
63 85
85 63
18 11
11 18
46 86
86 46
50 51
51 50
91 82
82 91
66 34
34 66
93 6
6 93
53 45
45 53
52 84
84 52
1 35
35 1
99 48
48 99
86 15
15 86
37 80
80 37
80 41
41 80
8 26
26 8
62 28
28 62
11 36
36 11
2 71
71 2
85 78
78 85
16 54
54 16
83 30
30 83
96 58
58 96
44 88
88 44
60 65
65 60
56 99
99 56
49 83
83 49
59 46
46 59
77 53
53 77
64 24
24 64
87 64
64 87
92 43
43 92
7 23
23 7
72 42
42 72
23 8
8 23
12 56
56 12
32 90
90 32
51 91
91 51
70 76
76 70
3 96
96 3
75 17
17 75
14 57
57 14
76 13
13 76
97 33
33 97
29 27
27 29
71 52
52 71
10 32
32 10
55 75
75 55
57 2
2 57
33 4
4 33
28 55
55 28
65 44
44 65
90 47
47 90
35 37
37 35
41 77
77 41
73 39
39 73
95 14
14 95
58 70
70 58
89 61
61 89
20 49
49 20
67 92
92 67
38 59
59 38
48 3
3 48
54 94
94 54
69 63
63 69
40 60
60 40
61 66
66 61
27 67
67 27
22 12
12 22
43 74
74 43
31 5
5 31
81 38
38 81
68 97
97 68
82 62
62 82
0 98
98 0
78 1
1 78
79 19
19 79
42 10
10 42
17 25
25 17
19 21
21 19
88 79
79 88
4 93
93 4
98 29
29 98
