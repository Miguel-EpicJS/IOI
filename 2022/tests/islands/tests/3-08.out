91041670623a69783795ed82bda33426101ce311
OK
1
132
180 196 120 166 152 94 170 6 162 80 134 78 192 184 190 14 150 84 74 2 174 154 86 38 56 12 98 186 124 104 136 30 142 16 144 114 128 68 122 50 24 40 106 42 178 64 132 126 112 188 20 176 118 130 194 46 4 36 66 8 18 72 26 27 158 159 27 26 159 158 72 18 8 66 36 4 46 194 130 118 176 20 188 112 126 132 64 178 42 106 40 24 50 122 68 128 114 144 16 142 30 136 104 124 186 98 12 56 38 86 154 174 2 74 84 150 14 190 184 192 78 134 80 162 6 170 94 152 166 120 196 180
